pages/blog/march-2023.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Community Update: March 2023"
+date: 2023-03-30T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2023-summary/blog-banner-march.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: 'Community update for March'
+featured: true
+---
+March came with some ecstatic news for the AsyncAPI community. We welcomed [Thulisile Sibanda](https://www.linkedin.com/in/v-thulisile-sibanda/) as our new Community Manager. And to top it all, our beloved community guardian, [Lukasz Gornicki](https://www.linkedin.com/in/lukasz-gornicki-a621914/) stepped up to become the new [Executive Director](https://github.com/orgs/asyncapi/discussions/615) for AsyncAPI.
+This is such an incredible milestone for the community and we can't wait to see the amazing things to follow in the next few months.
+
+## AsyncAPI Conf on Tour 2023
+One of our goals this year is to host [AACoT](https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1), a series of micro-conferences in different locations around the world. We are glad that our first location has been confirmed and we'll be in Madrid on the 18th - 19th of October 2023. [Call for speakers](https://sessionize.com/aacot-madrid) will be open soon and we are eagerly waiting to receive your submissions. For more details about other AACoT locations, keep an eye out for this [discussion](https://github.com/orgs/asyncapi/discussions/598).
+
+## Case Studies
+We recently added a new [case studies](https://www.asyncapi.com/casestudies) section on our website. It showcases real-world production scenarios and how different individuals adopt and integrate our specifications and tools. On this note, we are glad to have our first official case study from Adeo featured on the website. You can read more about it [here](https://www.asyncapi.com/casestudies/adeogroup).
```

Comment:
```suggestion
We recently added a new [case studies](https://www.asyncapi.com/casestudies) section on our website. It showcases real-world production scenarios and how different individuals adopt and integrate our specifications and tools. On this note, we are glad to have our first official case study from [Adeo](https://www.adeo.com/en/) featured on the website. You can read more about it [here](https://www.asyncapi.com/casestudies/adeogroup).
```

We are introducing our partners in a case study so let's remember to include a courtesy backlink.